doc/tech/LOGGING.adoc

```diff
@@ -0,0 +1,9 @@
+= Logging
+
+== API
+
+You can use different logging manage by spring profiles:
+
+* console with `logging-console` profile
```

Comment:
no need to add profile at end of your sentence 
* console with `logging-console`